app/views/search/_filters.html.haml

```diff
@@ -27,6 +27,9 @@
           = check_box_tag 'material_types[]', 'Interactive',  @form_model.material_types.include?('Interactive'),:id=>'Interactive'
           = label_tag 'interactive', t(:interactive).titleize
           %br
+          = check_box_tag 'material_types[]', 'Project',  @form_model.material_types.include?('Project'),:id=>'Project'
+          = label_tag 'project', t(:project).titleize
```

Comment:
I think this code means that the UI would have the word Project here. But I'd think we'd want it to read "Collection".

To simplify all of this, I think it'd be better to drop the use of "Project" in these changes. And just always refer to this as a collection. We aren't using the project model. We are just adding a new type of external activity. So calling it a collection everywhere in the code seems better.